app/src/main/java/com/nextcloud/talk/adapters/messages/PreviewMessageViewHolder.kt

```diff
@@ -228,6 +214,35 @@ abstract class PreviewMessageViewHolder(itemView: View?, payload: Any?) :
             true,
             viewThemeUtils!!
         )
+
+        // super.onBind is at this position, because:
+        // - it needs to be called after "placeholder" is set (otherwise placeholders are initially not loaded)
+        // - it needs to be before the show/hide logic is set (because super methods also have logic for this, which
+        // needs to be overwritten again)
+        super.onBind(message)
```

Comment:
This feels like a bit too finicky of a solution, but I guess it's fine in the interest of fixing the bug ASAP. At a minimum I would separate the logic to get the placeholder into its own method and call only that before `super.onBind` instead of all the rest.

I don't understand how this worked before with Fresco :\ It seems like the same problem would happen

Reply:
yes it's not a clean solution but it's gonna be a hotfix asap.
will extract the placeholder logic.

the "placeholder" solution was introduced when coil was introduced. but now i think of an easier solution. will have another look in the evening..

Reply:
for now i moved the logic for the placeholder to getPayloadForImageLoader(...)

this may still be not the best solution because the logic inside is also responsible for other things. 
but i think it's better than to call the super method in between other code which is now avoided.
see commit 4ccf8ac5a28e13a3a9f0095b7e0711251e876a4c
cc @AlvaroBrey